openedx/core/djangoapps/video_pipeline/api.py

```diff
@@ -9,7 +9,7 @@
 from oauth2_provider.models import Application
 from slumber.exceptions import HttpClientError
 
-from openedx.core.djangoapps.video_pipeline.config.waffle import ENABLE_VEM_PIPELINE, waffle_flags
+from openedx.core.djangoapps.video_pipeline.config.waffle import waffle_flags
```

Comment:
The way our diff-quality report works is it includes any errors from any lines you touched.  The quality errors are saying that waffe_flags also isn't used, so just delete the whole line.  See https://build.testeng.edx.org/job/edx-platform-quality-pipeline-pr/19473/Diff_20Quality_20Report/

Reply:
:ok_hand: